cupy/typing/_alias.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, TypeVar
+
+import numpy
+import numpy.typing as npt
+
+import cupy
+
+
+if numpy.lib.NumpyVersion(numpy.__version__) < '1.21.0':
+    raise ImportError('`cupy.typing` requires NumPy 1.21 or later.')
```

Comment:
Does it cost to support NumPy 1.20?

Reply:
If we support numpy<1.21 and python<3.9, we need to reimplement `GenericAlias` in CuPy's codebase.